website/pages/docs/concepts/policies.mdx

```diff
@@ -154,7 +154,7 @@ In addition, a `+` can be used to denote any number of characters bounded
 within a single path segment (this appeared in Vault 1.1):
 
 ```ruby
-# Permit reading the "teamb" path under any top-level path under secret/
+# Permit reading the "teamb" path under any top-level path under secret/.
```

Comment:
I think this is more confusing, and would not merge it.  "." in the context of a path means a reference to the folder itself, so I would read this (with the path listed below) as "secret/secret/+/teamb/" if the "." is there.